### ISSUE TITLE
Single clause doesn't work?

### ISSUE THREAD

**Issue by afhammad** [stoeffel/react-cond#20]
Is there a reason why a single clause renders nothing or is this a bug?

````javascript
<Cond value={x}>
   <Clause test={v => v}>
      ....
   </Clause>
</Cond>
````

**Comment by stoeffel** [comment]
I verified it. It's a bug (only when you use the component style).
Do you want to make a PR to fix this? Would be awesome.

**Comment by hemanth** [comment]
Nice catch!

**Comment by afhammad** [comment]
Sure, on it.

**Comment by stoeffel** [comment]
@afhammad awesome! thx

**Comment by afhammad** [comment]
@stoeffel can't get develop or test to run, they fail with:

````
const cacheA = testPerformanceCache && testPerformanceCache[a[0]];
        ^^^^^
SyntaxError: Use of const in strict mode.
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/ahmad/Projects/vendor/react-cond/node_modules/jest-cli/src/jest.js:13:18)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
````

also build doesn't actually compile the file, just copies it as-is. Is there a babel config missing?

**Comment by stoeffel** [comment]
Hmm. it works for me and it worked for @epeli. What node version do you have?

**Comment by afhammad** [comment]
upgrading node fixed the error!

**Comment by afhammad** [comment]
will this make it into a release soon?

**Comment by stoeffel** [comment]
Yop, I will publish soon.

**Comment by stoeffel** [comment]
Hopefully today :smile:

**Comment by afhammad** [comment]
Awesome, thanks.

**Comment by stoeffel** [comment]
@afhammad I published 0.3.1
